app/models/post.rb

```diff
@@ -1,3 +1,20 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Good job so far. It is advised to remove the commented codes here to keep your codebase neat 😃 

- [ ] There should be a `likes_counter` validation but it looks like you've commented it out, in this case I suggest that you uncomment it instead of cleaning it out